func/upgrade.sh

```diff
@@ -555,14 +555,30 @@ upgrade_b2_tool() {
 	fi
 }
 
+upgrade_nginx_resolver() {
```

Comment:
Is there a reason why we want to change it every update? One time should be enough I think

Reply:
Or only update for hestia-nginx?

Its configuration file will be overwritten after upgrade.

Reply:
hestia-nginx will get over written in that case we should update:
https://github.com/hestiacp/hestiacp/blob/main/src/deb/nginx/postinst

Only how to solve auto updates ... What if happens hestia-nginx is updated before hestia

Reply:
I'll revert this part of the changes first...